src/www/system_certmanager.php

```diff
@@ -1288,7 +1267,7 @@ function internalca_change() {
                     <textarea name="csr" id="csr" cols="65" rows="7"><?=$pconfig['csr'];?></textarea><br/>
                     <a href="#" class="csr_info_for_sign_csr btn btn-secondary"><?=gettext("Show Detail");?></a><br/>
                     <div class="hidden" data-for="help_for_csr_sign_csr">
-                      <?=gettext("Paste the CSR file here. (TODO: good English explanations. x509 extensions that OPNsense knows is automatically copied when clicking Next button and modifiable, and others are ignored)");?>
+                      <?=gettext("Paste the CSR file here.");?>
```

Comment:
It would be awesome if one could also upload a CSR file instead of pasting it :)

Reply:
probably, but I would focus on getting it functional and ready for release first.